.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [master]
   pull_request:
-    branches: [master]
```

Comment:
Should we update the version in `enterprise/__init__.py`? I see we did this in 1be2ff9547fc77732bc646bd46c0a467b5128fa8.

Reply:
Bumped the version and added the changelog in https://github.com/open-craft/edx-enterprise/pull/4/commits/5c2db6b70ef558d24b5add77e19ccdca69136781